lib/net.js

```diff
@@ -256,6 +258,9 @@ function Socket(options) {
 
   // Used after `.destroy()`
   this[BYTES_READ] = 0;
+
+  this[kLastTimeout] = 0;
+  this[kPrevTimeout] = 0;
```

Comment:
Why are these being attached here when they're used by the `http` module? It seems like these should instead be set by `http` so as not to affect (performance-wise) those who use `net` directly.